elasticsearch-api/utils/thor/templates/_documentation_top.erb

```diff
@@ -13,45 +13,45 @@
 # software distributed under the License is distributed on an
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 # KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
+# @specific language governing permissions and limitations
```

Comment:
This `@` is causing the failure in the License headers GitHub Action.

Reply:
fixed in https://github.com/elastic/elasticsearch-ruby/pull/2157/commits/25004b886758587ef2392495b80e7d1230830b52, thanks!